packages/tasks/src/tasks/text-generation/spec/input.json

```diff
@@ -1,94 +1,204 @@
 {
 	"$id": "/inference/schemas/text-generation/input.json",
 	"$schema": "http://json-schema.org/draft-06/schema#",
-	"description": "Inputs for Text Generation inference",
+	"description": "Text Generation Input",
 	"title": "TextGenerationInput",
 	"type": "object",
+	"required": ["inputs"],
 	"properties": {
 		"inputs": {
-			"description": "The text to initialize generation with",
-			"type": "string"
+			"type": "string",
+			"example": "My name is Olivier and I"
 		},
 		"parameters": {
-			"description": "Additional inference parameters",
-			"$ref": "#/$defs/TextGenerationParameters"
-		},
-		"stream": {
-			"description": "Whether to stream output tokens",
-			"type": "boolean"
+			"$ref": "#/$defs/TextGenerationInputGenerateParameters"
 		}
 	},
 	"$defs": {
-		"TextGenerationParameters": {
-			"title": "TextGenerationParameters",
-			"description": "Additional inference parameters for Text Generation",
+		"TextGenerationInput": {
+			"type": "object",
+			"required": ["inputs"],
+			"properties": {
+				"inputs": {
+					"type": "string",
+					"example": "My name is Olivier and I"
+				},
+				"parameters": {
+					"$ref": "#/$defs/TextGenerationInputGenerateParameters"
+				}
+			}
+		},
+		"TextGenerationInputGenerateParameters": {
 			"type": "object",
 			"properties": {
 				"best_of": {
 					"type": "integer",
-					"description": "The number of sampling queries to run. Only the best one (in terms of total logprob) will be returned."
+					"default": "null",
+					"example": 1,
+					"nullable": true,
+					"minimum": 0,
+					"exclusiveMinimum": 0
```

Comment:
😄 You got your input validation info in the end

Reply:
Ahah, yeah^^
But actually it's used no-where at the moment. The Python code just ignore those rules.